src/components/home/center/topHeader.tsx

```diff
@@ -4,10 +4,10 @@ import * as Wrapper from './style'
 import {Icon} from 'react-icons-kit'
 import {chevronRight} from 'react-icons-kit/feather'
 
-const TopHeader: FC<{id: number}> = ({id}) => {
+export const TopHeader: FC = () => {
   return (
     <Wrapper.Top>
-      <Wrapper.H1>{id}</Wrapper.H1>
+      <Wrapper.H1></Wrapper.H1>
```

Comment:
Is this needed?

Reply:
I needed it to add more alignement to flexbox.